packages/react/src/components/Label/Label.tsx

```diff
@@ -62,9 +62,9 @@ class Label extends UIComponent<WithAsProp<LabelProps>, any> {
   static propTypes = {
     ...commonPropTypes.createCommon({ color: true }),
     circular: PropTypes.bool,
-    icon: customPropTypes.itemShorthand,
+    icon: customPropTypes.itemShorthandWithoutJSX,
```

Comment:
Maybe we should rethink how we create these custom prop types - the user here must know that the image and icon are this special type...

Reply:
Can you please provide more info? We already have a custom propTypes for them...

Reply:
My point was, the user must know that the `Icon` and `Image` have different type then the other shorthands, and this is very prone to mistake in the future... Maybe we can make the component type param to the item shorthand and have one centralized place where this logic will be implemented. Other then this don't have any better idea.. What do you think?